README.md

```diff
@@ -118,6 +118,11 @@ are available
 * `CLAP_MANUAL_URL` and `CLAP_SUPPORT_URL` generate the urls in your description
 * `CLAP_MISBHEAVIOUR_HANDLER_LEVEL` can be set to `Terminate` or `Ignore` (default
   is `Ignore`) to choose your behaviour for a misbehaving host.
+* `CLAP_CHECKING_LEVEL` can be set to `None`, `Minimal`, or `Maximal` (default is
+  `Minimal`) to choose the error checking level for the plugin.
+* `CLAP_SMALLEST_ALLOWED_BLOCK_SIZE` can be set to any integer value to choose the
```

Comment:
So "smallest block size" is a bit of a misnomer, since it's possible for the plugin to end up processing blocks smaller than this. For example, let's say the user sets `CLAP_SMALLEST_ALLOWED_BLOCK_SIZE=32`, and then some host gives the plugin a buffer of 70 samples, with some events happening on every sample. The wrapper will process a block of 32, then another block of 32, but it will then be forced to process a block of 8 samples.

Any suggestions on what might be a better name?

Reply:
It really is the largest block size we will call you with. Perhaps use that name?

Reply:
Hmm, the problem is that it's not really the largest block size either. Like if the host gives the plugin a really big block with no events, the plugin will just process it all in one go.

i think maybe the most accurate name is something like `CLAP_EVENT_RESOLUTION_SAMPLES`. Basically "the plugin will handle events with this level of resolution", and if the host gives a weird block size (we may not see this until FL Studio comes along), we might get even better resolution for a few samples.